MAUI/ComboBox/Selection.md

```diff
@@ -9,6 +9,8 @@ documentation: ug
 
 # Selection in .NET MAUI ComboBox (SfComboBox)
 
+The ComboBox allows user to select single or multiple items from the drop-down list. The selection mode can be set by using the SelectionMode property. There are two different selection modes: Single, and Multiple.
```

Comment:
The ComboBox allows users to select single or multiple items from the drop-down list. The selection mode can be set by using the SelectionMode property. There are two different selection modes: Single and Multiple.